roulette/src/components/movieList/MovieList.js

```diff
@@ -0,0 +1,31 @@
+import './MovieList.css';
+import Movie from '../movieItem/Movie';
+
+const MovieList = ({ movies }) => {
+  return (
+    <>
+      {movies.length ? (
```

Comment:
Couple of points here:

1. Consider always creating a separate named function for method callbacks/handlers.
E.g. `list.map(mapHandler);`
2. Consider avoiding putting logic in JSX directly, use function calls or variable instead.
The difference between function call and variable would be the difference in placement relative to the component body itself: function calls for things from the outside and variable for things inside the body.
E.g. `const renderGenres = props.genres.map(renderGenre);`
`<> ... {renderGenres}... </>`